galaxy.yml

```diff
@@ -54,6 +54,8 @@ tags:
 # range specifiers can be set and are separated by ','
 dependencies:
   ansible.posix: "*"
+  community.postgresql: "*"
+  community.crypto: "*"
```

Comment:
Just make sure our documentation has any necessary updates. In case we tell people to manually install just ansible.posix .

Also, is this unrelated to the other change?

Reply:
when testing I noticed these dependencies weren't declared